README.md

```diff
@@ -60,6 +61,55 @@ Ensure that you meet the following prerequisites before using the extension:
 
 You successfully created a Zowe CLI `zosmf` profile. Now you can use all the functionalities of the extension.
 
+## Secure Credential Store
+
+You can now use Secure Credential Store plug-in in the Zowe Explorer extension. Secure Credentials Store plug-in lets you store your credentials securely. You can use the plug-in in the following ways:
```

Comment:
Use the Secure Credential Store Plug-in for Zowe CLI to store your credentials securely in Zowe Explorer in the settings file.

Reply:
That covers only 1 way of using the plug-in. The other one is SCS in ZE only (excluding CLI). If I replace what I have with what you suggest, I'll have to edit the rest of the doc and exclude the other way of using the plug-in.

Reply:
Okay, you can leave it for now. I guess my point is that the user won't really see either use case.